server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -494,22 +494,29 @@ private static void flushAll(final ClientContext context) {
     try {
       flusher.join(3000);
     } catch (InterruptedException e) {
-      // ignore
+      log.warn("Interrupted while waiting to join Flush thread");
     }
 
     while (flusher.isAlive() && System.currentTimeMillis() - start < 15000) {
       int flushCount = flushesStarted.get();
       try {
         flusher.join(1000);
       } catch (InterruptedException e) {
-        // ignore
+        log.warn("Interrupted while waiting to join Flush thread");
       }
 
       if (flushCount == flushesStarted.get()) {
         // no progress was made while waiting for join... maybe its stuck, stop waiting on it
         break;
       }
     }
+
+    flusher.interrupt();
+    try {
+      flusher.join();
+    } catch (InterruptedException e) {
+      log.warn("Interrupted while waiting to join Flush thread");
```

Comment:
I think it's a good idea to interrupt the flusher thread, if we're not going to wait on it anymore, but I still think the current thread needs to reset the interrupt flag on itself, when it is interrupted waiting for the join.

```suggestion
    flusher.interrupt();
    try {
      flusher.join();
    } catch (InterruptedException e) {
      Thread.currentThread.interrupt();
      log.warn("Interrupted while waiting to join Flush thread", e);
```